.codecov.yml

```diff
@@ -0,0 +1,24 @@
+coverage:
```

Comment:
Ok I see reconstructSPI is missing files like this that are present in the other repos. I assume you got them from the other ones and adapted anything that needed to be.

Would have been better to split into another PR (bite sized pieces!)... but let's go with it.